Dockerfile

```diff
@@ -36,34 +36,33 @@ COPY ./pkg/plugins/diff/delta_diff_server/Cargo.lock ./Cargo.lock
 COPY ./pkg/plugins/diff/delta_diff_server/Cargo.toml ./Cargo.toml
 
 # 3. Build only the dependencies to cache them in this layer
-RUN cargo build --release
+RUN RUSTFLAGS=-Ctarget-feature=-crt-static cargo build --release
 RUN rm src/*.rs
 
 # 4. Now that the dependency is built, copy your source code
 COPY ./pkg/plugins/diff/delta_diff_server/src ./src
 
 # 5. Build for release.
 RUN rm ./target/release/deps/delta_diff*
-RUN cargo build --release
+RUN RUSTFLAGS=-Ctarget-feature=-crt-static cargo build --release
 
 # lakectl image
-FROM --platform=$BUILDPLATFORM debian:11.6-slim AS lakectl
-RUN apt-get update
-RUN apt-get install -o APT::Keep-Downloaded-Packages=false -y ca-certificates
+FROM --platform=$BUILDPLATFORM alpine:3.16.0 AS lakectl
+RUN apk add -U --no-cache ca-certificates
```

Comment:
Not sure how this (and similar) lines should work: Docker itself keeps a cache, and will only re-run this line _if it or some line above it changes_.